composer.lock

```diff
@@ -1004,7 +1005,7 @@
                 "collections",
                 "iterator"
             ],
-            "time": "2017-07-22T10:37:32+00:00"
```

Comment:
@typhonius please update your local version of Composer: `composer global self-update`. It looks like you are using an older version that produces this deprecated timestamp format.